contract/src/main.rs

```diff
@@ -1161,8 +1175,39 @@ pub extern "C" fn transfer() {
         && utils::get_dictionary_value_from_key::<bool>(OPERATORS, &owner_operator_item_key)
             .unwrap_or_default();
 
+    let skip_checks = if let Some(contract) = utils::get_transfer_filter_contract() {
+        let mut args = RuntimeArgs::new();
+        args.insert(ARG_SOURCE_KEY, source_owner_key).unwrap();
+        args.insert(ARG_TARGET_KEY, owner).unwrap();
+
+        match &token_identifier {
+            TokenIdentifier::Index(idx) => {
+                args.insert(ARG_TOKEN_ID, *idx).unwrap();
+            }
+            TokenIdentifier::Hash(hash) => {
+                args.insert(ARG_TOKEN_ID, hash.clone()).unwrap();
+            }
+        }
+
+        let result = call_contract::<u8>(contract, TRANSFER_FILTER_METHOD, args);
```

Comment:
@darthsiroftardis @deuszex Hi, this is the only remaining comment for me on this PR, I guess the method should be an arg too

Otherwise I'm approving the PR.

thanks

Reply:
Is this still relevant, it seems outdated?

Reply:
@gRoussac Since this time we are the ones who "allow" others to be called back we are fine with the static "burned in" entry_point name